operations/research-and-development/product/technical-writing-team-handbook/README.md

```diff
@@ -213,41 +206,50 @@ For best results, please include as much content, context, and detail as possibl
 
-#### Help Wanted tickets/doc issues
+### Help Wanted tickets/doc issues
 
 Help Wanted tickets are generally not linked to a release and are more flexible in terms of timeline and delivery date. You can find the documentation Help Wanted tickets [here](https://github.com/mattermost/docs/labels/Help%20Wanted). To start working on one, assign it to yourself, add a comment indicating you’ll be working on it, find the relevant document in the [source](https://github.com/mattermost/docs/tree/master/source) directory, read through the [README](https://github.com/mattermost/docs/blob/master/README.md) file if you're not familiar with the process, and get started.
 
 If you have any questions, you can post them in the [Documentation](https://community.mattermost.com/core/channels/documentation) channel and we'll be happy to help.
 
-#### Engineering/developer documentation
+### Engineering/developer documentation
 
 Most, if not all, contributions to the Mattermost project have a documentation impact. As part of the development and submission process, it’s recommended that the relevant documentation be updated \(or created\) and included in the PR. This provides consistency and accuracy in communicating the changes/new feature and cuts down on having multiple issues and PRs for related documentation.
 
 Visit the [Contribute to Documentation](https://handbook.mattermost.com/operations/operations/publishing/publishing-guidelines/voice-tone-and-writing-style-guidelines/submitting-documentation-with-your-pr) page to get started with submitting documentation with your PR. You can read more about the review process [here](https://developers.mattermost.com/contribute/getting-started/code-review).
 
+## PR tips and best practices
+
+* Try submit files in batches rather than single PRs per file. This reduces the PR load and also groups your changes more effectively.
+* When there are lots of changes requested, especially on a docs PR where the changes are "safe" (meaning they don't need to be QA tested), you can use the "Add suggestion to batch" feature in GitHub to commit all of the suggestions at once. You have to be in the "Files changed" tab to actually use the feature, though it is also shown in the "Conversation" tab. Then you can use one commit message like "Added batch suggestions from @Carrie Warner". This just makes the commit history of the PR a lot more readable and useful.
+* Descriptive PR titles and commit messages go a long way, especially when the PR is referenced in the future. The default PR titles that GitHub gives us like "Updated README.md" are not very helpful because there's no way to know the purpose of the PR by reading the title. For all the reviewer knows, maybe the whole file was translated to Japanese because we realize that's our main audience, but I wouldn't know if that was the case unless it was in the PR title.
+
+## GitHub training
+
+GitHub training videos coming soon!
```

Comment:
Should we have a ticket for this? Maybe we should have a ticket for now instead of this point here, because we don't know where they will exist or if they will be made in the format of videos, for instance.

Also, I think the point about descriptive PR titles and commit messages would also be very helpful to developer contributions, as well as the upcoming GitHub training information. I think this information should be readily available to all kinds of contributors, and so it may be missed if it's on this page. Also, I don't want these tips to be missed in general because they're buried on the bottom of this page.

Reply:
Another route we could take right away is finding some good quality, reputable GitHub training material online and providing links to that, rather than creating our own. GitHub itself may have such material we could point to. @aspleenic and the Community Team may also have additional resources they could recommend.

Reply:
That's a great question, @mickmister! And I noticed that Carlos +1ed the idea about GH training videos. It is something I planned to raise in the Handbook Champions sync - but as the Champions pool is currently small, I can raise an issue in an appropriate repo and crowdsource some help.

Reply:
I agree with @cwarnermm that a quick action would be to link to some GitHub documentation such as https://docs.github.com/en/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request.

Creating videos on the topic of contributing, catered to our projects specifically, sounds very helpful to newcomers. As far as mentioning the videos here right now, 0/5 I think posting links to GitHub docs is good for now.

What do you think about the second part of my comment above pertaining to surfacing these tips to be more visible than the bottom of this page, with some of the tips also placed on developer docs?